### PR TITLE
feat: g2-oidc OpenStack cluster with OIDC support added, README added

### PR DESCRIPTION
**Pull request by freznicek**
This PR introduces:
 * README.md
 * G2 OpenStack cluster with OIDC support and external ceph connection
 * Git repository manifests are now *-gitrepository.yaml instead of ambiguous *-repository.yaml

### CODE REVIEW

File: apps/g2-oidc/03-openstack/03-openstack.base/05-rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+
```
Comment (chnyda) [inline apps/g2-oidc/03-openstack/03-openstack.base/05-rabbitmq.yaml]: 2 useless empty lines.

File: apps/g2-oidc/03-openstack/03-openstack.base/08-glance.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        node_selector_key: openstack-control-plane
+        node_selector_value: enabled
+      job: *common-labels
+      registry: *common-labels
+    #pod:
+    #  replicas:
+    #    api: 3
+    #    registry: 2
+    # NOTE(portdirect): glance APIv1 is required for heat in Newton
+
```
Comment (chnyda) [inline apps/g2-oidc/03-openstack/03-openstack.base/08-glance.yaml]: too many empty lines

File: apps/g2-oidc/03-openstack/03-openstack.base/05-rabbitmq.yaml (hunk 1 above)
Comment (freznicek) [reply apps/g2-oidc/03-openstack/03-openstack.base/05-rabbitmq.yaml]: All unneeded empty lines in k8s/CRD manifests except empty lines in raw encoded config files were dropped by 04a287d016f855e488ed0d5ac6fc917c145f7c0c.

File: apps/g2-oidc/03-openstack/03-openstack.base/08-glance.yaml (hunk 2 above)
Comment (freznicek) [reply apps/g2-oidc/03-openstack/03-openstack.base/08-glance.yaml]: All unneeded empty lines in k8s/CRD manifests except empty lines in raw encoded config files were dropped by 04a287d016f855e488ed0d5ac6fc917c145f7c0c.

**Pull request merged**